src/basic/cgroup-util.c

```diff
@@ -2588,10 +2588,16 @@ int cg_enable_everywhere(CGroupMask supported, CGroupMask mask, const char *p) {
                         continue;
 
                 n = cgroup_controller_to_string(c);
+
```

Comment:
nitpick: Avoid adding blank line here. Not really necessary.

Reply:
Will remove.